pkgs/development/tools/build-managers/cmake/2.8.nix

```diff
@@ -56,24 +71,24 @@ stdenv.mkDerivation rec {
     (concatMap (p: [ (p.dev or p) (p.out or p) ]) buildInputs);
 
   configureFlags = [
-    "--docdir=/share/doc/${name}"
+    "--docdir=/share/doc/cmake-${version}"
```

Comment:
Still doesn't look exactly the same as before because `pname` can be `cmake-cursesUI` or `cmake-qt4UI`.

Reply:
sorry